pint/compat/__init__.py

```diff
@@ -127,7 +127,8 @@ def _to_magnitude(value, force_ndarray=False):
 try:
     import pandas as pd
     HAS_PANDAS = True
-    HAS_PROPER_PANDAS = hasattr(pd.core.arrays.base, "ExtensionOpsMixin")
+    HAS_PROPER_PANDAS = hasattr(pd.core, "arrays") and \
+                        hasattr(pd.core.arrays.base, "ExtensionOpsMixin")
 except ImportError:
```

Comment:
can you change to 

```python
HAS_PROPER_PANDAS = (
        hasattr(pd.core, "arrays") 
        and hasattr(pd.core.arrays.base, "ExtensionOpsMixin")
    )
```

please to avoid the `\` continuation

Reply:
Sure thing. Done and pushed.

Reply:
my bad, I think the code review failed because of indenting. I suspect it should be 

```python
try:
    import pandas as pd
    HAS_PANDAS = True
    HAS_PROPER_PANDAS = (
        hasattr(pd.core, "arrays")
        and hasattr(pd.core.arrays.base, "ExtensionOpsMixin")
    )
except ImportError:
HAS_PROPER_PANDAS = HAS_PANDAS = False
```

Reply:
I'm not sure... the indentation shouldn't make a difference with the brackets right? The build actually passed. Only the coverage review complains about decreased coverage... What's this about?

Reply:
The code still works but bors failed it for code reviews which I think is because my review has still not been switched. I'll fix that now, but that indentation looks awkward so if you have 3 minutes to fix that would be great (but as you say not essential and probably better done with an automated linter in future).

The coverage thing is weird and we don't have a solution yet, see https://github.com/lemurheavy/coveralls-public/issues/1190

Reply:
Alright. Done.